### PR TITLE
Proof Reading

### PR DESCRIPTION
**Pull request by ropery**

### CODE REVIEW

**Comment by acabal** [comment]
Hi there, have you confirmed these changes against the Victoria edition we're basing this ebook on?

When it comes to Shakespeare, there are a lot of different version and opinions, so in order to avoid an eternity of people suggesting their pet changes back and forth, we want to base these editions strictly on Victoria. So, we only want to accept actual transcription typos and not much else.

**Comment by ropery** [comment]
The vast majority of those are transcription errors, and a few obvious errors in the Victoria edition.

The transcription errors I corrected include punctuation and spelling and missing words, but also e.g. the expansion of "Serv.", where is should be "Servingman" instead of "Servant".

Errors in the Victoria edition include the lowercase "lent" and "pentecost" (while many other lowercasing such as "love" & "death" when referring to anthropomorphic entities are left alone, though they probably should be capitalized), spelling errors like "villanous" and "plaintain".

In the reading process I compared the Victoria edition to the Folger Shakespeare, and mainly the New Cambridge Shakespeare edition, which has very helpful notes. I ensure you that I have been very conservative about accepting any changes to the Victoria edition. Where I do introduce a change, the reason is explained in the commit message. The change "bird -> bud" for example, is made because "bird" simply makes no sense and "bud" makes every sense, and probably a simple error in the Victoria edition.

You can see for yourself with `git log -p --word-diff` that all changes are minor; I have refrained from any substantial changes, and whenever there is room for alternatives I don't presume to make a change.

**Comment by ropery** [comment]
There are 11 commits in this PR: the last one contains the most necessary changes (straight-up errors); the other ten commits are made separately because each requires some explanation.

No change is my "pet change" I don't think, but still true sticklers/dogmatists may yet object to any change at all, despite proof that the Victoria edition contains straight-up errors.

If you find my changes reasonable and conservative enough, then why not go ahead and merge this PR? We can always revert any changes if anyone cares enough and has reason enough to do so.

**Comment by ropery** [comment]
Come on now, what will you do?

By the way, is there a way to build the epub (etc.) files from the source files online? maybe sort of like a building service? (I haven't tried to install the se tools locally, what with the apparently heavy dependencies.)

Or if not & if you won't merge this, perhaps I might trouble you to build an epub from my source for me & post it here as an attachment? (I would like to reap the fruit of my own labor, as I'm sure you understand.) Thank you.

**Comment by acabal** [comment]
Relax - this is my hobby and my day is busy. I'll get to this soon.

File: src/epub/text/act-3.xhtml [hunk 1]
```diff
@@ -375,7 +375,7 @@
 							<td/>
 							<td epub:type="z3998:verse">
 								<p>
-									<span>Now, Tybalt, take the villain back again,</span>
```
Comment (acabal) [inline src/epub/text/act-3.xhtml:378]: Revert

File: src/epub/text/act-3.xhtml [hunk 2]
```diff
@@ -2381,7 +2381,7 @@
 							<td epub:type="z3998:persona">Capulet</td>
 							<td epub:type="z3998:verse">
 								<p>
-									<span>And why, my lady wisdom? hold your tongue,</span>
```
Comment (acabal) [inline src/epub/text/act-3.xhtml:2384]: Revert

File: src/epub/text/act-4.xhtml [hunk 3]
```diff
@@ -247,7 +247,7 @@
 									<br/>
 									<span>God join’d my heart and Romeo’s, thou our hands;</span>
 									<br/>
-									<span>And ere this hand, by thee to Romeo seal’d,</span>
```
Comment (acabal) [inline src/epub/text/act-4.xhtml:250]: Revert

File: src/epub/text/act-4.xhtml [hunk 4]
```diff
@@ -1291,7 +1291,7 @@
 						</tr>
 						<tr>
 							<td epub:type="z3998:persona">Peter</td>
-							<td>Musicians, O, musicians, “Heart’s ease, Heart’s ease:” O, an you will have me live, play “Heart’s ease.”</td>
```
Comment (acabal) [inline src/epub/text/act-4.xhtml:1294]: Revert

File: src/epub/text/act-5.xhtml [hunk 5]
```diff
@@ -81,7 +81,7 @@
 								<p>
 									<span>Then she is well, and nothing can be ill:</span>
 									<br/>
-									<span>Her body sleeps in Capel’s monument,</span>
```
Comment (acabal) [inline src/epub/text/act-5.xhtml:84]: Revert

File: src/epub/text/act-5.xhtml [hunk 6]
```diff
@@ -99,7 +99,7 @@
 							<td epub:type="z3998:persona">Romeo</td>
 							<td epub:type="z3998:verse">
 								<p>
-									<span>Is it even so? then I defy you, stars!</span>
```
Comment (acabal) [inline src/epub/text/act-5.xhtml:102]: Revert

File: src/epub/text/act-5.xhtml [hunk 7]
```diff
@@ -785,7 +785,7 @@
 									<br/>
 									<span>To grubs and eyeless skulls? as I discern,</span>
 									<br/>
-									<span>It burneth in the Capel’s monument.</span>
```
Comment (acabal) [inline src/epub/text/act-5.xhtml:788]: Revert

File: src/epub/text/act-5.xhtml [hunk 8]
```diff
@@ -1389,7 +1389,7 @@
 									<br/>
 									<span>For I will raise her statue in pure gold;</span>
 									<br/>
-									<span>That while Verona by that name is known,</span>
```
Comment (acabal) [inline src/epub/text/act-5.xhtml:1392]: Revert

**Comment by acabal** [comment]
OK, I've had some time to look at this.

I appreciate the effort and the interest in updating Shakespeare. However, most of these changes bring the text away from Victoria and towards other editions. We specifically want to avoid that situation because any work on Shakespeare is by its nature highly scholarly and opinionated. We don't have the resources to do a serious editorial edition of any Shakespeare, and Shakespeare deserves a scholar's full attention to really create a good edition - as opposed to SE and its contributors, who are just a bunch of internet strangers volunteering piecemeal, without any particularly deep Shakespeare knowledge.

We also don't want to host a blend of editions either, because then people looking at the Victoria page scans will become confused as to what's a typo in our edition, vs. what's an editorial choice from a later commit. And then I spend the rest of my life ping-ponging between people trying to bring it back to Victoria, and people trying to add random editorial decisions from other editions. 

Not only that, but it looks like you copied and pasted a good chunk of copyrighted explanatory material from other editions into these commit messages, which we cannot have in this repo under any circumstances. This repo, including its commit history, must be totally US-PD.

So with that in mind, we can keep:

- 'plantain' dd345c6b8e7ac6b9d37a76e174b09771e99532e7
- 'censorship' d24252daeb1f645bcc04b1ca12e39883a463e2b8
- *Most* of 'sundry fixes' 8725fde126de0ff41d20e05b0049a4bab9409c61 as most of that commit is transcription fixes; but see the review for some lines to revert.

I'll be happy to accept this PR if you can make those updates by rebasing the other commits out. (We need them dropped using rebase and not just reverted with new commits, because of the copyrighted material in the commit messages.)

I know it can be frustrating to have to live with what might be a less-perfect edition. But considering SE's long-term maintenance goals, the complexity of Shakespeare, and the careful scholarship it deserves, that's the situation we're in right now.

**Review by acabal — CHANGES_REQUESTED** [review]

**Pull request closed without merging**